cpp/benchmarks/streaming/ndsh/q09.cpp

```diff
@@ -467,7 +467,7 @@ rapidsmpf::streaming::Node sort_by(
     auto chunk =
         rapidsmpf::ndsh::to_device(ctx, msg.release<rapidsmpf::streaming::TableChunk>());
     auto table = chunk.table_view();
-    auto rounded = cudf::round(
+    auto rounded = cudf::round_decimal(
```

Comment:
you can't just do this because it requires the input column to be decimal already which it is not

Reply:
Ah, yeah, looks like I was mistaken. 
https://docs.rapids.ai/api/libcudf/legacy/group__transformation__unaryops#ga0c3cc731433773ab9e36aca8d0d25912